README.MD

```diff
@@ -54,4 +54,4 @@ we think our users will want to know about.  Also, we **never** distribute our m
 If you are a contest administrator setting up PC2, we recommend joining the PC2 announcement list so that you are notified 
 if significant issues which might affect the integrity of your contest are found.
 
-Please see [this page](http://hera.ecs.csus.edu/mailman/listinfo/pc2-announce "Join the Pc2 Mailing list") to join the PC2 mailing list.
\ No newline at end of file
+Please see [this page](http://hera.ecs.csus.edu/mailman/listinfo/pc2-announce "Join the Pc2 Mailing list") to join the PC2 mailing list.
```

Comment:
First, thanks so much for taking time to review things.  Welcome!
On this line change:  what am I missing?  I don't see a single character difference between the deleted line and the added line (?)

Reply:
I have my editor set to make sure every file has a newline at the end so that’s what you’re seeing.

The reason is that it prevents conflicts